.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -40,7 +39,7 @@ Before opening your pull request:
   - This is **required** for all bug fixes and new features.
 - [ ] I have updated the project documentation, if applicable.
   - This is **required** for new features.
-- [ ] If this is my first contribution, I have added myself to the list of contributors.
+- [ ] If this is my first contribution, I have added myself to the [list of contributors](../doc/markus-contributors.txt).
```

Comment:
You've identified the right text, but the link can't be a relative link (since on GitHub the URL is not the same as your local current working directory). Instead, use an absolute URL *to the file on GitHub* (on the MarkUsProject/MarkUs repository).